src/V8JsiRuntime.cpp

```diff
@@ -446,9 +446,11 @@ void V8Runtime::createHostObjectConstructorPerContext() {
       isolate_, constructorForHostObjectTemplate->GetFunction(context_.Get(isolate_)).ToLocalChecked());
 }
 
+std::once_flag g_tracingInitialized;

```

Comment:
>g_tracingInitialized [](http://example.com/codeflow?start=15&length=20)

(nit) should this be declared inside of ifdef _WIN32 clause?

Reply:
Thanks, good catch.